search-app/docs/api.md

```diff
@@ -1686,33 +1687,46 @@ Find collections matching service concept id.
 ##### <a name="c-polygon"></a> Polygon
 
 Polygon points are provided in counter-clockwise order. The last point should match the first point to close the polygon. The values are listed comma separated in longitude latitude order, i.e. lon1, lat1, lon2, lat2, lon3, lat3, and so on.
+This parameter supports the and/or option as shown below.
 
     curl "%CMR-ENDPOINT%/collections?polygon=10,10,30,10,30,20,10,20,10,10"
 
+    curl "%CMR-ENDPOINT%/collections?polygon=10,10,30,10,30,20,10,20,10,10&polygon=11,11,31,11,31,21,11,21,11,11&options[polygon][or]=true"
+
 ##### <a name="c-bounding-box"></a> Bounding Box
 
 Bounding boxes define an area on the earth aligned with longitude and latitude. The Bounding box parameters must be 4 comma-separated numbers: lower left longitude, lower left latitude, upper right longitude, upper right latitude.
+This parameter supports the and/or option as shown below.
 
     curl "%CMR-ENDPOINT%/collections?bounding_box=-10,-5,10,5
 
+    curl "%CMR-ENDPOINT%/collections?bounding_box=-10,-5,10,5&bounding_box=-11,-6,11,6&options[bounding_box][or]=true
```

Comment:
With the addition of spatial options, should we drop the individual shape options now? I don't feel strongly against this, but concerned that we are offering too many options that are probably too confusing.

Reply:
I figured I would just leave them in if for some reason a user wanted to fine-tune their search more, and the work is already done